omnibus/config/software/python3.rb

```diff
@@ -14,7 +14,7 @@
   dependency "libyaml"
 
   source :url => "https://python.org/ftp/python/#{version}/Python-#{version}.tgz",
-         :sha256 => "c7cfa39a43b994621b245e029769e9126caa2a93571cee2e743b213cceac35fb"
+         :sha256 => "015115023c382eb6ab83d512762fe3c5502fa0c6c52ffebc4831c4e1a06ffc49"
```

Comment:
`default_version` still `3.8.1` on line 3 above. That should be updated.

Also, this is only bumping the python version on linux/macOS and not on windows which would happen below and I believe is a pre-baked built.

Reply:
I just saw the previous review, think we'd typically want windows and linux to be aligned, we can discuss. In any case this is broken as it currently stands as `version` will make us download a different artifact that will not meet the specified hash.

Reply:
Yeah I bumped it down to `3.8.1` because I tried only updating unix, but that didn't work. I'll revert the `default_version` back to `3.8.5` due to what you already mentioned.

Reply:
Also agree that they should be aligned, not even sure the script supports separate versions currently.